### ISSUE TITLE
Reduced isohemagglutinin level 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5611]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0032139

**Suggested revision of textual definition**

Change label to Partial absence of isohemagglutinins

**Comment by pnrobinson** [comment]
Also change Absent isohemagglutinin level => Complete or near-complete absence of isohemagglutinins

**Comment by pnrobinson** [comment]
Adding as synonym
